components/script/dom/htmlelement.rs

```diff
@@ -44,27 +45,30 @@ pub struct HTMLElement {
 }
 
 impl HTMLElement {
-    pub fn new_inherited(tag_name: LocalName, prefix: Option<Prefix>,
-                         document: &Document) -> HTMLElement {
+    pub fn new_inherited(tag_name: LocalName, prefix: Option<Prefix>, document: &Document) -> HTMLElement {
```

Comment:
This file contains a lot of modifications caused by your editor automatically running rustfmt on the file, which makes it harder to review. Could you disable this feature, revert the changes to the file, and make your modifications to it again?

Reply:
This does not appear to have been addressed.